docs/spectator/core/meters/timer.md

```diff
@@ -3,8 +3,9 @@
 A Timer is used to measure how long (in seconds) some event is taking. Two types of Timers
 are supported:
 
-* `Timer`: for frequent short-duration events.
-* `LongTaskTimer`: for long-running tasks.
+* `Timer` - for frequent short-duration events.
+* `LongTaskTimer` - for long-running tasks.
+* [Percentile Timer] - for tracking the frequency of measurements.
```

Comment:
"tracking frequency" sounds more like a counter. I think something like "if percentile approximations are needed in addition to basic stats".

Also, percentiles are an extension to a short-duration timer. For this page I think it is probably better to focus on the short and long use-cases. The short duration section can discuss basic timers and then link off to bucket and percentile extensions. Querying is also different between short/long timers.

Reply:
Ack. Folks reading this page will expect some mention of percentiles, hence the link out. I could drop it at the bottom and tell them to take a look if these don't fit their use case.

Do you have an example of the long timer query patterns?